upi/openstack/down-06_load-balancers.yaml

```diff
@@ -0,0 +1,16 @@
+# Required Python packages:
+#
+# ansible
+# openstacksdk
+
+- import_playbook: common.yaml
+
+- hosts: all
+  gather_facts: no
+
+  tasks:
+  - name: 'Remove the cluster load balancers'
```

Comment:
This playbook does not have a corresponding "create" counterpart. Can you document its purpose in the README?